### PR TITLE
Describe how multipath conn are currently proxied

### PR DESCRIPTION
**Pull request by boucadair**
Brief overview of how TCP connections are proxied.

### CODE REVIEW

**Review by SpencerDawkins — APPROVED** [review]
I saw a "rulse" that's probably "rules".

I know ASCII art is painful, but it would be easier to understand if the protocol ladder diagram put each message on its own line, so that the ordering is more obvious :-)

Otherwise, this looks helpful. Thanks for submitting it.

File: draft-quic-atsss-reqs.mkd [hunk 1]
```diff
@@ -120,7 +120,7 @@ The two scenarios that we consider in this document are:
 
 ## MPTCP-Proxied Connections
 
-This section describes how TCP connections matching ATSSS rulse are currently established over differnt access networks.
+This section describes how TCP connections matching ATSSS rules are currently established over differnt access networks.
```
Comment (nkeukeleire) [inline draft-quic-atsss-reqs.mkd]: "differnt" to "different"

Comment (boucadair) [reply draft-quic-atsss-reqs.mkd]: Fixed in 43fb73c. Thanks

File: draft-quic-atsss-reqs.mkd [hunk 2]
```diff
@@ -118,7 +118,34 @@ The two scenarios that we consider in this document are:
...
+          |        ACK, MPC        |          ACK           |
+          |          ...           |           ...          |
+
+Legend:
+     []: Convert Protocol TLVs
+    MPC: MP_CAPABLE option [RFC8684] 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+{: #fig-example title="An Example of MPTCP-proxied Connection Matching an ATSSS Rule."}
+
+This approach provides 0-RTT (Zero Round-Trip Time) conversion service since no extra delay is induced by the Convert protocol compared to connections that are not proxied. Also, the Convert Protocol does not require any encapsulation (no tunnels, whatsoever).
```
Comment (mpiraux) [inline draft-quic-atsss-reqs.mkd:188]: I see this as a good requirement for ATSSS, your thoughts ?

File: draft-quic-atsss-reqs.mkd [hunk 3]
```diff
@@ -118,7 +118,34 @@ The two scenarios that we consider in this document are:
...
+
+The ATSSS-capable UE sends packets bound to connections matching an ATSSS rule to the provisioned Transport Converter and destination port number. Concretely, the UE initiates the MPTCP connection towards the Transport Converter and indicates the IP address and port number of the Server within the connection establishment packet (i.e., in the payload of the SYN sent to the Transport Converter).  Doing so enables the Transport Converter to immediately initiate a connection towards that Server, without experiencing an extra delay. The Transport Converter waits until the receipt of the confirmation that the Server agrees to establish the connection before confirming it to the Client.
+
+A flow example of an MPTCP-proxied connection is show in {{fig-example}}. This example assumes the Server is not MPTCP-aware. The access network used to establish the matching ATSSS rule to select the access network to be used to establish the first subflow. The instruction included in the ATSSS rule will be followed for the management of the MPTCP connection.
```
Comment (mpiraux) [inline draft-quic-atsss-reqs.mkd]: > The access network used to establish the matching ATSSS rule to select the access network to be used to establish the first subflow.

This sentence is not correct.

Comment (mpiraux) [reply draft-quic-atsss-reqs.mkd]: > The instruction included in the ATSSS rule will be followed for the management of the MPTCP connection.

I am new to the ATSSS spec: Is an ATSSS rule equivalent to a particular steering mode ?

Comment (boucadair) [reply draft-quic-atsss-reqs.mkd]: "Steering Mode" is a mandatory attribute of an ATSSS rule.

Comment (boucadair) [reply draft-quic-atsss-reqs.mkd]: > 
> 
> > The access network used to establish the matching ATSSS rule to select the access network to be used to establish the first subflow.
> 
> This sentence is not correct.

Fixed in d2d4c90

File: draft-quic-atsss-reqs.mkd (hunk 2 above)
Comment (boucadair) [reply draft-quic-atsss-reqs.mkd:188]: Note that for the tunneling part, this can be understood as an key requirement only if you are familiar with #17

**Pull request merged**